### DISCUSSION TITLE
Commands Required to Update to new version of nominatim

### DISCUSSION THREAD

**Discussion by br4v0** [osm-search/Nominatim#2134] [Q&A]
Hello Guys,
I am a novice in docker and nominatim. I have installed OSM on my AWS server but i need a set of right commands to update OSM and specific commands on the region basis. I want to update my OSM for Asia and America but i can't find out the right way to do it. Kindly let me know what are the exact commands to perform this operation.

**Comment by mtmail** [comment]
Docker specific support requests should be asked in https://github.com/mediagis/nominatim-docker (assuming that’s the steps you took to install the initial version). List which version you’re currently running and if the question is about updating the software or data. With data it helps to list data size when the last update was, as sometimes it’s faster to do a complete reinstall instead of applying hundreds of small updates.

**Comment by lonvia** [comment]
https://nominatim.org/release-docs/latest/admin/Advanced-Installations/ has information how to install and update multiple extracts in the same Nominatim database.
